website/components/temporary_text-split/style.module.css

```diff
@@ -37,7 +38,7 @@
 }
 
 .content {
-  width: 50%;
+  width: 46%;
```

Comment:
These couple of style tweaks were necessary to ensure spacing between the `text` and `content` sides of this component.

Also chose to leave this `temporary_` component in place - an alternative would be to use `TextSplit`'s `content` prop, but refactoring didn't feel worth it, and felt like it's fine to wait for a revised design.